cdap-cli/src/main/java/co/cask/cdap/cli/command/HelpCommand.java

```diff
@@ -67,4 +73,93 @@ public String getPattern() {
   public String getDescription() {
     return "Prints this helper text";
   }
+
+  /**
+   * Prints the given command pattern which wrap at the given column width. It prints on multiple lines like this:
+   *
+   * <pre>{@code
+   * command sub-command <arg1> <arg2>
+   *                     <arg3> [<optiona-long-arg4>]
+   * }
+   * </pre>
+   *
+   * @param pattern the command pattern to print
+   * @param output the {@link PrintStream} to write to
+   * @param colWidth width of the column
+   */
+  private void printPattern(String pattern, PrintStream output, int colWidth) {
+    if (pattern.length() <= colWidth) {
+      output.println(pattern);
+      return;
+    }
+
+    // Find the first <argument>, it is used for alignment in 2nd line and onward.
+    int startIdx = pattern.indexOf('<');
+    // If no '<', it shouldn't reach here (they should be short command). If it does, just print it.
```

Comment:
If no '<', it shouldn't reach here (it should be a short command). If it does, just print it. \[correct?]